docs/adr/0002-use-react-redux-and-typescript-for-subscription-management-pages.md

```diff
@@ -0,0 +1,129 @@
+# Use React, Redux, and Typescript for subscription management pages
+
+* Deciders: Ben Bangert, Ian Bicking, Les Orchard
+* Date: 2019-05-08
+
+## Context and Problem Statement
+
+A solution for isolating third-party payment widgets from the rest of account
+management can include building a separate web app on its own dedicated domain.
+As a side effect in building that app, we have the opportunity to choose
+technologies for building it that don't necessarily follow the rest of FxA.
+
+## Decision Drivers
+
+* Opportunity for a fresh start with tech stack without rewriting
```

Comment:
I think this has been a decision driver, whether it's a good idea in & of itself